crypto/mem_sec.c

```diff
@@ -266,7 +266,7 @@ static size_t sh_getlist(char *ptr)
     for (; bit; bit >>= 1, list--) {
         if (TESTBIT(sh.bittable, bit))
             break;
-        OPENSSL_assert((bit & 1) == 0);
+        assert((bit & 1) == 0);
```

Comment:
Without having closer look, some OPENSSL_asserts in memory allocator are very likely to be totally justified. Because that's where you can be actually pretty sure of memory corruption. Let's make @richsalz say something explicit...